app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,9 @@
         </div>
       </div>
       <% if current_user %>
+        <small><%= menu_link_to "My Profile", user_path(current_user) %></small>
+        <small><%= menu_link_to "Friend Requests", friendships_path %></small>
+        <small><%= menu_link_to "Signed in as: #{current_user.name}", users_path %></small>
         <%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
       <% else %>
         <%= link_to 'Sign in', user_session_path %>
```

Comment:
- [ ] Instead of having the if/else statement to create a helper method that has the if/else and renders a partial that contains the links within the if/else statement instead of having the logic in the view